pyproject.toml

```diff
@@ -18,17 +18,21 @@ include = [
 [tool.poetry.plugins."nautobot.workers"]
 "clear" = "nautobot_chatops.workers.clear:clear"
 "nautobot" = "nautobot_chatops.workers.nautobot:nautobot"
+"ansible" = "nautobot_chatops.integrations.ansible.worker:ansible"
 
 [tool.poetry.dependencies]
 # TODO Move to ~ requirements instead of ^
 python = "^3.7"
 Markdown = "!=3.3.5"
 PyJWT = "^2.1.0"
+PyYAML = "*"
```

Comment:
I don't think we should be allowing all versions of PyYAML, but I would be curious to hear what others would say?

Also, shouldn't these three go in the extras only? If they are in the [tool.poetry.dependencies] it will still install them.

Reply:
Will fix extra dependencies to be compatible with https://python-poetry.org/docs/pyproject/#extras

Reply:
Fixed.